README.md

```diff
@@ -330,6 +333,94 @@ RUN go build -o hello hello.go && \
     xx-verify hello
 ```
 
+## Rust
+
+Building Rust can be achieved with the `xx-cargo` wrapper that automatically
+sets up the target triple and also `pkg-config` and C compiler.
+
+The wrapper supports rust installed via [`rustup`](https://rustup.rs/)
+(alpine/debian), distribution packages (alpine/debian) and the [official `rust` image](https://hub.docker.com/_/rust).
+
+### Building on Alpine
+
+```dockerfile
+# official rust image
+FROM --platform=$BUILDPLATFORM rust:alpine
```

Comment:
Is this any different from `FROM alpine && RUN apk add rust`?

Reply:
Yes, the official image ships `rustup` for both alpine and debian: https://github.com/rust-lang/docker-rust/blob/47a11a42c5a321d688fae525eaa7f1cf9940198b/1.64.0/alpine3.16/Dockerfile